experiment/test_config.yaml

```diff
@@ -760,8 +760,8 @@ images:
     - --gcp-node-image=gci
   cosbeta:
     args:
-    - --image-family=cos-beta
-    - --image-project=cos-cloud
+    - --env=KUBE_GCE_NODE_IMAGE=cos-only-for-testing-63-10032-39-0
+    - --env=KUBE_GCE_NODE_PROJECT=gke-node-images
```

Comment:
why we need to switch from flags to global envs?

Reply:
`kubernetes_e2e.py` doesn't support `--image` with `--image-project` yet.

Reply:
https://github.com/kubernetes/test-infra/issues/3320